doc/release-notes/release-8.sgml.in

```diff
@@ -30,9 +30,19 @@ The Squid-@SQUID_RELEASE@ change history can be <url url="https://github.com/squ
 
 <p>The most important of these new features are:
 <itemize>
-	<item>
+	<item>Helper binary changes
```

Comment:
We have discussed this before: This list title promises new features. Removal of a helper is not a new feature.

I also disagree that a removal of a supposedly rarely used helper qualifies for "the most important" change classification.

```suggestion
	<item>
```